app/models/manageiq/providers/amazon/cloud_manager/auth_key_pair.rb

```diff
@@ -1,6 +1,4 @@
 class ManageIQ::Providers::Amazon::CloudManager::AuthKeyPair < ManageIQ::Providers::CloudManager::AuthKeyPair
-  AwsKeyPair = Struct.new(:name, :key_name, :fingerprint, :private_key)
```

Comment:
Was key_name just not used?

Reply:
So I'm guessing they created `AwsKeyPair` to match the Openstack object as closely as possible, but key_name was not used by https://github.com/ManageIQ/manageiq/pull/21759/files#diff-b80f64b28488206bf8d41b9b53fb77a5490267a1a91b5d1e4f0897af88951554L43-L46 so I didn't include it.  This is now just returning the params for creating of the record